events-api/src/modules/tickets/dto/requests/payment.dto.ts

```diff
@@ -0,0 +1,25 @@
+import { IsInt, IsNumber, IsPositive, IsString, MaxLength, MinLength } from "class-validator"
+import { TimeFormat } from "src/shared/constants/time.constant"
+import { IsDateFormatValid } from "src/shared/validators/date-format.validator"
+
+export class PaymentDto {
+    @IsInt()
+    @IsPositive()
+    last4Digits: number
```

Comment:
Do we want to check if the length of this field is 4 digits?

Reply:
created a validator isFourDigits